src/Commands/BatchProcess.php

```diff
@@ -212,7 +213,13 @@ protected function processBatch(BeamType $type): int
                         'idempotency_key' => Str::uuid()->toString(),
                     ]);
                     BeamBatch::where('id', $batchId)->update(['transaction_id' => $transaction->id]);
-                    BeamBatchTransactionCreated::safeBroadcast($param['beamId'], $param['collectionId'], $transaction->id);
+                    BeamBatchTransactionCreated::safeBroadcast(
```

Comment:
Why is this modified? The `BeamBatchTransactionCreated::construct` params didn't change though?